src/test/java/RozetkaSergey/Vijar.java

```diff
@@ -0,0 +1,60 @@
+package RozetkaSergey;
+
+import ClasesToAllUs.TestInit;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+public class Vijar extends TestInit {
+
+String login = "grinred69@gmail.com";
+String pass = "23112010vi";
+
+    @Test
+    public void loginTest() {
+        enterToGoogle();
+        sleep(1);
```

Comment:
sleep it's bad practice. Best Explicit Wait

Reply:
ok I will do it tommorow

Reply:
ok)